consensus/hotstuff_leader.go

```diff
@@ -189,19 +210,26 @@ func (handler *HotstuffLeaderMessageHandler) HandleCommitMessage(m *consensusMod
 	handler.emitTelemetryEvent(m, msg)
 
 	if err := handler.anteHandle(m, msg); err != nil {
-		m.nodeLogError(typesCons.ErrHotstuffValidation.Error(), err)
+		m.logger.Error().Err(err).Msg(typesCons.ErrHotstuffValidation.Error())
 		return
 	}
 
 	if err := m.didReceiveEnoughMessageForStep(Commit); err != nil {
-		m.nodeLog(typesCons.OptimisticVoteCountWaiting(Commit, err.Error()))
+		m.logger.Info().Msg(typesCons.OptimisticVoteCountWaiting(Commit, err.Error()))
 		return
 	}
-	m.nodeLog(typesCons.OptimisticVoteCountPassed(m.height, Commit, m.round))
+
+	m.logger.Info().Fields(
+		map[string]any{
+			"height": m.height,
+			"round":  m.round,
+			"step":   Prepare,
```

Comment:
Ditto https://github.com/pokt-network/pocket/pull/420#discussion_r1090059855

Reply:
s/Prepare/Commit

This is inside the `HandleCommitMessage` function, so I think we should be logging `"step": Commit` instead of `"step": Prepare` which we do  in the `HandlePrepareFunction`